sources/assets/Stride.Core.Assets/AssetReferenceDataSerializer.cs

```diff
@@ -15,6 +15,13 @@ public override void Serialize(ref AssetReference assetReference, ArchiveMode mo
     {
         if (mode == ArchiveMode.Serialize)
         {
+            if (assetReference == null)
+            {
+                stream.Write(Guid.Empty);
+                stream.Write("");
```

Comment:
Writing an empty string is a no-op isn't it? You probably wanted to match the other two lines, but I don't think it is necessary.

Reply:
Changes in AssetReferenceDataSerializer were deleted, even if the exception was thrown there initially.